token/program-2022/src/extension/mod.rs

```diff
@@ -398,37 +429,25 @@ pub trait BaseStateWithExtensions<S: BaseState> {
             Ok(adjust_len_for_multisig(total_len))
         }
     }
+    /// Calculate the new expected size if the state allocates the given number
+    /// of bytes for the given fixed-length extension type.
+    fn try_get_new_account_len<V: Extension + Pod>(&self) -> Result<usize, ProgramError> {
+        try_get_new_account_len_for_extension_len::<S, V>(
+            self.get_tlv_data(),
+            pod_get_packed_len::<V>(),
+        )
+    }
 
     /// Calculate the new expected size if the state allocates the given number
-    /// of bytes for the given extension type.
-    ///
-    /// Provides the correct answer regardless if the extension is already present
-    /// in the TLV data.
-    fn try_get_new_account_len<V: Extension + VariableLenPack>(
+    /// of bytes for the given variable-length extension type.
```

Comment:
nit: (also get rid of the last word on the previous line)
```suggestion
    /// variable-length extension instance.
```